src/bitmovinCastManager.ts

```diff
@@ -43,6 +43,8 @@ export const BitmovinCastManager = {
    * This method needs to be called before `Player` creation to enable casting features.
    * If no options are provided, the default options will be used.
    *
+   * IMPORTANT: This should only be called when the Google Cast SDK is available in the application.
+   *
```

Comment:
@strangesource I added this as on iOS `initializeCasting` crashes the app when Cast SDK is not in the app. How is it on Android?

Reply:
Same on Android, it crashes as well.